larq_compute_engine/mlir/python/converter_test.py

```diff
@@ -10,12 +10,20 @@
     "larq_compute_engine.tflite.python.interpreter_wrapper_lite"
 ] = mock.MagicMock()
 
+import larq_compute_engine
 from larq_compute_engine.mlir.python.converter import convert_keras_model
 from larq_compute_engine.mlir._graphdef_tfl_flatbuffer import (
     convert_graphdef_to_tflite_flatbuffer as mocked_converter,
 )
 
 
+def test_version():
+    assert (
+        hasattr(larq_compute_engine, "__version__")
+        and "." in larq_compute_engine.__version__
+    )
```

Comment:
@lgeiger is this test failing because we don't actually install the package, so setup.py isn't being run?

Reply:
Yes we don't actually install LCE in the Python only tests since it would require us to build all the C++ dependencies.

Reply:
Makes sense, then I think I'll just remove the test because I don't think there's much extra value including it.